examples/dashboard/dashboard.ipynb

```diff
@@ -0,0 +1,242 @@
+{
```

Comment:
<p>The thing about reading from github URLs here is that its hard-coded to the branch name, so when the branch is merged/deleted the URL won't work. Isn't this effectively the same as reading from s3?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/saturncloud/examples/pull/102/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Yeah, I'm not thrilled with that tbh, but I think once this is approved we can hardcode it to `main`.

Reply:
URLs from s3 are like https://s3-us-east-2.amazonaws.com/saturn-public-data/nyc taxi/data/dashboard/pickup_grouped_by_zone.csv